velox/exec/Operator.cpp

```diff
@@ -379,14 +379,18 @@ void Operator::recordSpillStats() {
     lockedStats->addRuntimeStat(
         kSpillReadTimeUs,
         RuntimeCounter{
-            static_cast<int64_t>(lockedSpillStats->spillReadTimeUs)});
+            static_cast<int64_t>(lockedSpillStats->spillReadTimeUs) *
+                Timestamp::kNanosecondsInMicrosecond,
+            RuntimeCounter::Unit::kNanos});
   }
 
   if (lockedSpillStats->spillDeserializationTimeUs != 0) {
     lockedStats->addRuntimeStat(
         kSpillDeserializationTimeUs,
```

Comment:
Should the name still have Us at the end?

Reply:
I rename it to follow convention. thanks!